src/App.tsx

```diff
@@ -1,24 +1,29 @@
 import React from 'react';
 import './App.scss';
+import Clock from './components/clock/Clock';
 
-const App: React.FC = () => {
-  const timerId: NodeJS.Timer = setInterval(() => {
-    const date: Date = new Date();
+class App extends React.Component {
+  state = {
+    isClockVisible: true,
+  };
 
-    // eslint-disable-next-line
-    console.log(date.toLocaleTimeString());
-  }, 1000);
+  startClock = () => {
+    this.setState({ isClockVisible: true });
+  };
 
-  return (
-    <div className="App">
-      <h1>React clock</h1>
-      <p>
-        Current time:
-        {' '}
-        {/* Print the time here instead of DevTools */}
-      </p>
-    </div>
-  );
-};
+  stopClock = () => {
+    this.setState({ isClockVisible: false });
+  };
+
+  render() {
+    return (
+      <div className="App">
+        {this.state.isClockVisible && <Clock />}
+        <button type="button" onClick={this.stopClock} className="button__red">Hide clock</button>
+        <button type="button" onClick={this.startClock} className="button__green">Show clock</button>
```

Comment:
длинные строки, добавляй переносы
```suggestion
        <button
          type="button"
          onClick={this.stopClock}
          className="button__red">
            Hide clock
        </button>
```